readthedocs.yml

```diff
@@ -1,5 +1,6 @@
 conda:
-  file: .rtd-environment.yml
+  file: test-environment.yml
 
 python:
    setup_py_install: true
+   version: 3.5
```

Comment:
Any reason to tie this to 3.5 rather than 3.6? Does it need to be tied to a version at all?

Reply:
3.6 failed in read the docs. Let's leave at 3.5 for now and we can look at it in the future.